src/Blueprint/Blueprint/Cmdlets/BlueprintDefinition/PublishAzureRMBlueprint.cs

```diff
@@ -31,10 +34,10 @@ public class PublishAzureRmBlueprint : BlueprintDefinitionCmdletBase
         [ValidateNotNullOrEmpty]
         public string Version { get; set; }
 
-        // To-Do: ChangeNotes will be added in the next SDK release
-        /* [Parameter(ParameterSetName = ParameterSetNames.PublishBlueprint, Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "To-Do")]
+        [Parameter(ParameterSetName = ParameterSetNames.PublishBlueprint, Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = ParameterHelpMessages.ChangeNotes)]
+        [ValidateLength(0,500)]
         [ValidateNotNullOrEmpty]
-        public string ChangeNotes { get; set; }*/
+        public string ChangeNotes { get; set; }
```

Comment:
@filizt since we need to avoid plural parameter names in PowerShell, we should rename this to something like `-ChangeContent` or `-ChangeRecord` (since `-ChangeNote` doesn't really make sense).

Also, since we're adding a new parameter, you will want to regenerate the markdown help for this cmdlet.

Reply:
I ended up making it ChangeNote since everywhere else we're calling it ChangeNotes and not calling it something else would not make senses. Let me know if this works.